src/objc/objects.rs

```diff
@@ -259,3 +260,30 @@ impl super::ObjC {
         mem.free(object.cast());
     }
 }
+
+/// Checks if an instance of the provided class responds to a selector (has a method in it's class
+/// chain) with the given name.
+///
+/// Note that this only checks instances of the class, not the class itself.
+/// This means that class methods of the provided class are not checked, only instance methods.
+pub fn instances_respond_to_selector(
+    env: &mut Environment,
```

Comment:
Hmm… I think this gets quite complicated when we try to use the same naming that Objective-C does, and it's even a bit misleading since it doesn't take a `SEL` like its Objective-C counterpart. What if `class_has_method` had a sibling method in the same file, named something like `class_has_method_named`? That would be a better fit for the current organisation of things. The description would only need to say that it does the same thing as `class_has_method`, except it doesn't panic or do an allocation when the selector is unknown. I guess `object_has_method` could be the variant that does the `read_isa` for you.

Also FWIW you could use the new `let … else` syntax in this method body to simplify it slightly, if you wanted.